calico_containers/tests/st/test_multi_host.py

```diff
@@ -24,7 +24,7 @@ def test_multi_host(self):
         host1.listen(calicoctl % ("node --ip=%s" % host1_ip))
         host2.listen(calicoctl % ("node --ip=%s" % host2_ip))
 
-        calico_port = "DOCKER_HOST=localhost:2377"
+        calico_port = "export DOCKER_HOST=localhost:2377;"
```

Comment:
Actually, I was having problems with the `VAR=thing cmd` syntax, and this was my overkill way to fix it and stop worrying. But either way, I overhaul this in aa1-convert-tests, so I'd rather leave it as is here and move further discussion on it to that branch.